Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts

```diff
@@ -68,6 +70,10 @@ function getLastSuccessful(serverEndpointUrl: string, jobName: string, cred: Cre
     return defer.promise;
 }
 
+function AddIssue(message) {
```

Comment:
Why not use tl.warning() from vsts-task-lib/task?

Reply:
Fixed.